src/intl.ts

```diff
@@ -121,6 +121,12 @@ export const commonMessages = defineMessages({
   cancel: {
     defaultMessage: "Cancel",
   },
+  preferences: {
+    defaultMessage: "Preferences",
+  },
```

Comment:
```suggestion
    defaultMessage: "Preferences",
    description: "language preferences"
  },
```

Reply:
Adding description in the dictionary will be helpful, but perhaps "user preferences modal" would be better because there is a plan to manage not only language setting here (region (country) selection, etc.). Don't you agree? Maby you have a better proposition?

Reply:
I agree :)